decisions/0001-bundle-granularity.rst

```diff
@@ -154,7 +243,7 @@ To re-use a Unit, you would first make a Link to the Bundle where that Unit came
 from, and then specify the Link prefix before the directory where the Unit comes
 from::
 
-    <xblock-include definition="arch_hangout_videos/video+blockstore_overview"/>
+    <xblock-include definition="arch_hangout_videos/video/blockstore_overview"/>
```

Comment:
Note that another option we could use is

```
<xblock-include source="arch_hangout_videos" definition="video/blockstore_overview" />
```

which I think is more future-proof. But I'm fine with either, and went with what's in the doc here for now.

Reply:
Yeah, I like your version better, explicitly splitting out the source Link name.